src/components/LoadingBar/LoadingBar.cy.tsx

```diff
@@ -0,0 +1,60 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    LoadingBar,
+    LoadingBarSize,
+    LoadingBarState,
+    loadingBarSizeClasses,
+    loadingBarStateClasses,
+} from './LoadingBar';
+import React from 'react';
```

Comment:
Move the 'react' import to the top of the file.

Reply:
fixed